vars/testNamespace.groovy

```diff
@@ -0,0 +1,5 @@
+import io.openshift.Utils;
```

Comment:
I don't know the reason for this if it used somewhere. Maybe it is required for Jenkins file